### PR TITLE
60835_create_course

### PR DESCRIPTION
**Pull request by ahammad-reyad**
Course created

### CODE REVIEW

File: app/helpers/admin/courses_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module Admin::CoursesHelper
```
Comment (dieunb) [inline app/helpers/admin/courses_helper.rb]: remove this file

File: app/views/admin/courses/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                <%= f.label :end_date, class: "col-sm-2 control-label" %>
+                <div class="col-sm-10">
+                  <%= f.date_field :end_date, class: "form-control",
+                    value: Time.now.strftime("%Y-%m-%d") %>
+                </div>
+              </div>
+              <div class="form-group">
+                <%= f.label :status, class: "col-sm-2 control-label" %>
+                <div class="col-sm-10">
+                  <%= f.select :status, ["Not started", "Started", "Finish"], {}, {class: "form-control"} %>
```
Comment (dieunb) [inline app/views/admin/courses/new.html.erb]: i18n

File: app/controllers/admin/courses_controller.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
+class Admin::CoursesController < ApplicationController
+  load_and_authorize_resource
+  before_action :load_subjects, only: [:new, :edit]
+
+  def new
+  end
+
+  def create
```
Comment (dieunb) [inline app/controllers/admin/courses_controller.rb]:
```
+  def create
 +    if @course.save
 +      flash[:success] = t "course.create_success"
 +    else
 +      flash[:danger] = t "course.create_failure"
 +    end
 +    redirect_to root_url
 +  end
```

Comment (ahammad-reyad) [inline app/controllers/admin/courses_controller.rb]: updated

**Pull request merged**